gordo_components/builder/mlflow_utils.py

```diff
@@ -398,48 +401,36 @@ def get_spauth_kwargs() -> dict:
     )
 
 
-class DatetimeEncoder(json.JSONEncoder):
+class MetadataEncoder(json.JSONEncoder):
     """
     Encode datetime.datetime objects as strings
 
     Example
     -------
-    >>> s = json.dumps({"now":datetime.now(tz=UTC)}, cls=DatetimeEncoder, indent=4)
+    >>> s = json.dumps({"now":datetime.now(tz=UTC)}, cls=MetadataEncoder, indent=4)
     >>> s = '{"now": "2019-11-22 08:34:41.636356+"}'
     """
 
+    @staticmethod
+    def is_int_or_float(obj):
+        return any([type_str in str(type(obj)) for type_str in ("float", "int")])
+
+    @staticmethod
+    def reduce_bit_type(obj):
+        for type_cls, type_str in [(float, "float"), (int, "int")]:
+            if type_str in str(type(obj)):
+                return type_cls(obj)
+
```

Comment:
We certainly don't want this... 

What I _think_ you're after is basically checking if the number object is a numpy instance which is not json serializable. ie.
```python
a = numpy.int32(1) 
json.dumps({"my-number": a})  # This will fail.
```

So what I think you want, is more something like a numpy to python conversion:
```
if isinstance(a, np.float):
    return float(a)
elif isinstance(a, np.integer):
    return int(a)
else:
    return a
```

Reply:
That's correct-a-mundo. Do you think there may be other float/int types outside of numpy that might not be covered by `json`?